### PR TITLE
Fix integration tests settings.gradle

### PR DESCRIPTION
**Pull request by wu-hui**
`new File()` works from the gradle installation path, switching to `file()` for project path.

Also some more .gitignore entries.

### CODE REVIEW

File: gma/integration_test/settings.gradle [hunk 1]
```diff
@@ -36,4 +36,4 @@ if (!(new File(firebase_cpp_sdk_dir)).exists()) {
             firebase_cpp_sdk_dir))
 }
 gradle.ext.firebase_cpp_sdk_dir = "$firebase_cpp_sdk_dir"
-includeBuild "$firebase_cpp_sdk_dir"
+includeBuild "$firebase_cpp_sdk_dir"
```
Comment (DellaBitta) [inline gma/integration_test/settings.gradle:39]: nit: add a new line here.

File: installations/integration_test/settings.gradle [hunk 2]
```diff
@@ -36,4 +36,4 @@ if (!(new File(firebase_cpp_sdk_dir)).exists()) {
             firebase_cpp_sdk_dir))
 }
 gradle.ext.firebase_cpp_sdk_dir = "$firebase_cpp_sdk_dir"
-includeBuild "$firebase_cpp_sdk_dir"
+includeBuild "$firebase_cpp_sdk_dir"
```
Comment (DellaBitta) [inline installations/integration_test/settings.gradle:39]: nit: add a new line here.

**Review by DellaBitta — DISMISSED** [review]
Approved, but can you fix the end of file new lines?

File: gma/integration_test/settings.gradle (hunk 1 above)
Comment (wu-hui) [reply gma/integration_test/settings.gradle:39]: Done.

File: installations/integration_test/settings.gradle (hunk 2 above)
Comment (wu-hui) [reply installations/integration_test/settings.gradle:39]: Done.

**Review by DellaBitta — DISMISSED** [review]

**Review by a-maurice — APPROVED** [review]

**Pull request merged**